src/sass/components/_footer.scss

```
@@ -0,0 +1,114 @@
+.footer {
+  padding: 60px 0;
+
+  @include respond-to(phone-and-down) {
+    padding-bottom: 30px;
+  }
+
+  &-top {
+    padding-bottom: 60px;
+    border-bottom: 1px solid $line-gray;
+
+    @include respond-to(phone-and-down) {
+      padding-bottom: 30px;
+    }
+
+    &__list {
+      @include flex(row, flex-start, space-between);
+      gap: 70px;
+      flex-wrap: wrap;
+
+      @include respond-to(tablet-and-down) {
+        gap: 30px;
+      }
+      @include respond-to(phone-and-down) {
+        flex-direction: column;
+      }
+    }
+
+    &__item {
+      display: flex;
+      flex-direction: column;
+
+      &:first-child {
+        padding: 0 !important;
+      }
+
+      h2 {
+        margin-bottom: 40px;
+      }
+      p,
+      a,
+      i,
+      span {
+        font-size: 13px;
+        line-height: 20px;
+        color: $p-gray;
+        font-weight: 500;
+      }
+      a {
+        margin-bottom: 15px;
+        font-weight: 500;
+      }
+      span {
+        margin-left: 10px;
+        font-family: $roboto;
+      }
+      .desc {
+        margin-bottom: 20px;
+      }
+      .detail {
+        line-height: 24px;
+        margin-bottom: 10px;
+      }
+
+      @include respond-to(phone-and-down) {
+        h2 {
+          margin-bottom: 20px;
+        }
+        .desc,
+        a {
+          margin-bottom: 10px;
+        }
+      }
+    }
+
+    &__title {
+      font-size: 16px;
+      text-transform: uppercase;
+    }
+  }
+
+  &-payment-support {
+    margin-top: 30px;
+    p {
+      font-size: 13px;
+      line-height: 20px;
+      color: $p-gray;
+      margin-bottom: 20px;
+    }
+    .payment {
+      padding: 5px;
+    }
+    .payment img {
+      @include box(120px, 40px, 0);
```

```suggestion
      @include box(120px, 40px);
```
radius có giá trị mặc định là 0 rồi thì k cần truyền nữa. Ngoài ra thì trong cái mixin đó thêm 1 đoạn if vào để tránh bị dư thừa `border-radius: 0`